libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.ReplaceDialog.schema

```diff
@@ -2,21 +2,21 @@
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
     "type": "object",
-    "title": "Replace Dialog",
-    "description": "This is a action which replaces the current dialog with the target dialog",
+    "title": "Replace dialog",
+    "description": "Replace current dialog with another dialog.",
     "properties": {
         "dialog": {
             "$type": "Microsoft.IDialog",
-            "title": "Dialog",
-            "description": "This is the dialog to switch to."
+            "title": "Dialog name",
+            "description": "Current dialog will be replaced by this dialog."
         },
         "options": {
             "type": "object",
-            "title": "Options binding",
-            "description": "Bindings to configure the options object to pass to the dialog.",
+            "title": "Value",
+            "description": "One or more values that are passed to the replacing dialog.",
```

Comment:
>One or more values that are passed to the replacing dial [](start = 28, length = 56)

Defines the options object passed to the dialog that is being started.

Reply:
What is the reason behind suggesting this specific update?
Here are a few things we've kept in mind with all of our updates:
1. (try to) not repeat the word we're describing in the explanatory sentence.

- Avoiding the word 'option', using street language to be more accessible than strictly coding language.

2. As much as possible, use accessible (street vs coding) language to explain things
- 'options object' may be codewise exactly what it is, but both words have other meanings in non-coding context which can cause confusion. 

3. As short but specific as possible
-    (ie in this case we know the context is Replace dialog, this is more informative than 'the dialog being started'. Also there could be confusion about the ReplacING or 'ReplacED' dialog so being specific is important here.


So - with this, @tomlm and @vishwacsena I'd prefer to leave the proposed copy as is. Would you be ok with that?